test/e2e/config/docker.yaml

```diff
@@ -58,25 +58,34 @@ providers:
       new: --metrics-addr=:8080
     files:
     - sourcePath: "../data/shared/v1.0/metadata.yaml"
-  - name: v1.2.11 # supported release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
```

Comment:
Should remove the 1.2 releases from this so we're only covering supported branches.

Reply:
So I also update the `- name: v1.3.6` on line 70?